tests/test_computations.py

```diff
@@ -0,0 +1,59 @@
+"""Test the function of computations module."""
+
+import numpy as np
+import pytest
+
+from wristpy.core import computations
```

Comment:
Minor point: Any reason why you import some things by module (e.g. computations) and some by class/function (e.g. Measurement)?

I'd keep this consistent across your code; common in most Python open source is importing the class/function, my preference follows the [google python style guide](https://google.github.io/styleguide/pyguide.html#22-imports) where you only import modules (with a few exceptions). 

Pick whichever you prefer, but I'd pick one and make sure you and @frey-perez are consistent on this.

Reply:
Assuming this isn't currently consistent across the codebase, an easy transition method is that if your PR affects an old file, you're responsible for correcting this in _that_ file.